src/main/java/org/fundacionjala/coding/dennis/cipher/Cipher.java

```diff
@@ -1,25 +1,27 @@
-package org.fundacionjala.coding.dennis;
+package org.fundacionjala.coding.dennis.cipher;
 
 /**
  *
  */
-public interface Encrypto {
+public interface Cipher {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)